src/utils/index.js

```diff
@@ -169,7 +167,6 @@ export const extractToLib = file => {
   } else {
     rimraf.sync(extractDir);
     fs.mkdirSync(extractDir);
-    return Promise.resolve(extractDir);
```

Comment:
We don't want to return here b/c we've just `rimraf`'d. We need to extract so that `copyToBin` has files to copy